projects/izumi/index.js

```diff
@@ -15,6 +33,15 @@ async function pool2() {
 }
 
 module.exports = {
-  fetch,
-  pool2
+    ethereum: {
+        fetch: fetchEthereum,
+    },
+    polygon: {
+        fetch: fetchPolygon,
+    },
+    arbitrum: {
+        fetch: fetchArbitrum,
+    },
+    pool2,
```

Comment:
can this be changed to 
pool2: {
 fetch: pool2
}
please

Reply:
Done~